src/v3.rs

```diff
@@ -217,6 +217,17 @@ impl Sender {
         self.host = host.into();
     }
 
+    /// Sets client to use for the API. This is useful if you want to customize the client.
+    pub fn set_client(&mut self, client: Client) {
```

Comment:
Maybe it would be better to allow `new` to accept an `Option<Client>` and just unwrap it or use a default one.

Reply:
@gsquire 

Is this what you meant? https://github.com/gsquire/sendgrid-rs/pull/114/commits/b5ea71350dc8bea470c36e765fee263ddff0aa83

Reply:
Exactly. We'll need a `new_blocking` function as well so users can configure that client. Otherwise this LGTM.

Reply:
Added! @gsquire 

https://github.com/gsquire/sendgrid-rs/pull/114/commits/62358ce9320534db0cb8c98ceefcb73f4d833275